css/src/components/input.css

```diff
@@ -124,10 +124,31 @@
 
 .Input-icon--right .Icon {
   cursor: pointer;
-  margin-left: var(--spacing);
   color: var(--inverse-lighter);
+  border-radius: 10px;
+}
+
+.Input-icon--right:focus-visible .Icon {
+  outline: var(--spacing-s) solid var(--secondary-shadow);
+}
+
+.Icon:focus-visible {
```

Comment:
Please don't apply any class to `Icon` directly. This will affect the all other components where `Icon` component is used